Docs: Moving IAM docs

urbiz-grafana opened the pull request:
As discussed.

Checks in https://github.com/grafana/website/pull/27769.

lwandz13 committed:
Merge branch 'docs/sql-expressions-updates-082025' of github.com:grafana/grafana into docs/sql-expressions-updates-082025

(cherry picked from commit b0bfbbf547cf8a5ed8ac07e7f252f0503794a624)

wbrowne committed:
Plugins: StaticFS should implement FSRemover (#110706) (#110943)

make staticfs implement fs removal interface

lwandz13 committed:
created new doc

imatwawana committed:
[release-12.2.0] Docs: Add panel filtering (#110960)

Co-authored-by: Sam Jewell <2903904+samjewell@users.noreply.github.com>

imatwawana committed:
[release-12.2.0] Docs: Replace screenshots for pill and markdown cells (#110962)

imatwawana committed:
[release-12.2.0] docs: clarify that data links must use variable names, not labels (#110965)

Co-authored-by: Sidharth Chauhan <chauhansiddharth71@gmail.com>

imatwawana committed:
[release-12.2.0] Docs: ad hoc filter improvements (#110961)

Commit:
[release-12.2.0] Alerting: Fix bug where rules with identical mute/active intervals produced conflicting routes (#110971)

Alerting: Fix bug where rules with identical mute/active intervals produced conflicting routes (#110935)

Alerting: Fix hash collision in NotificationSettings fingerprint
(cherry picked from commit fc3636acf26d8e44bff3f6ac04f5e10190f1ea11)

Co-authored-by: Alexander Akhmetov <me@alx.cx>

kminehart committed:
[release-12.2.0] CI: fix bump version action to use grafana-delivery-bot (#110976) (#110981)

CI: fix bump version action to use grafana-delivery-bot (#110976)

* update bump-version

* Add id-token: write

* update generate-token step

* pull-requests -> pull_requests

* clone with token and set right name

(cherry picked from commit c28a9178716cd5026884456354468970798e5ad8)

fastfrwrd committed:
[release-12.2.1] Table: Restore previous footer behavior of reducers applying to filtered data (#111041) (#111109)

Table: Restore previous footer behavior of reducers applying to filtered data (#111041)

* Table: Restore previous footer behavior of reducers applying to filtered data

* update e2e to match new behavior

(cherry picked from commit f258d8a41726e7bb482ae1cf8549a2686bfdb38c)

fastfrwrd committed:
[release-12.2.1] Table: Fix logic to calculate footer height (#110954) (#111107)

Table: Fix logic to calculate footer height (#110954)

* Table: Fix logic to calculate footer height

* add non-numeric footer case to gdev

* Update packages/grafana-ui/src/components/Table/TableNG/utils.ts



* Update packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx



---------


(cherry picked from commit cb37539ed7ca14230fda6e61012ab519eabccab7)

Co-authored-by: Leon Sorokin <leeoniya@gmail.com>

Commit:
[release-12.2.1] Fix: Fix redirection after login when Grafana is served from subpath (#111069)

Fix: Fix redirection after login when Grafana is served from subpath (#110889)

Fix short link (/goto) redirection when Grafana is served from subpath

(cherry picked from commit ccc87a03f0ec70eb28f9ce49221834718abeca7d)

Co-authored-by: Misi <mgyongyosi@users.noreply.github.com>

ashharrison90 committed:
Geomap: Only prefix with grafana public path if relative url (#111081) (#111106)

only prefix with grafana public path if relative url

samsch committed:
[release-12.2.1]: Accessibility: enable responsive reflow of variables in dashboard edit (#111033)

Accessibility: enable responsive reflow of variables in dashboard edit (#110967)

enable responsive reflow of variables in dashboard edit

adela-almasan committed:
[release-12.2.1]  Actions: Add permission check to missing panels (#111103)

Commit:
[release-12.2.1] Chore: bump `axios` to a version without CVE (#111110)

Chore: bump `axios` to a version without CVE (#111076)

bump axios to a version without CVE

(cherry picked from commit 7bba15141667636ebeaad2bff3b641b799c07ed7)

Co-authored-by: Ashley Harrison <ashley.harrison@grafana.com>

Commit:
[release-12.2.1] LDAP: Restore test user mapping functionality (#111120)

LDAP: Restore test user mapping functionality (#110841)

* Migrate LdapPage from connect() to React-Redux hooks

* Convert LDAP debug page into a drawer and hook it into settings

* prettier

* Use the Text component and make the input and button look like they do on the main settings page.

* Bring back isLoading and put in a LoadingPlaceholder

* i18n-extract

* rejigger

* linter fix

(cherry picked from commit 585b53bc7dba1925926a95e86aa8f1d45a2e74d6)

Co-authored-by: John Troy <jtroy@users.noreply.github.com>

Commit:
[release-12.2.1] Chore: Improve short url redirection (#111183)

Chore: Improve short url redirection (#111162)

Improve short url redirection

(cherry picked from commit 81fe57478fa5bc0977d7c00b62848485fc3ae35c)

Co-authored-by: Misi <mgyongyosi@users.noreply.github.com>

kminehart committed:
[release-12.2.1] backport bump-version.yml and release-build.yml (#111341)

* [release-12.2.1] backport bump-version.yml and release-build.yml

* add release-npm.yml

Commit:
[release-12.2.1] CI: Fix NPM workflow inputs (#111350)

Fix referring to inputs (#111345)

(cherry picked from commit 1d6c1da94f34266b7477af1bd9fc05b4c237caf4)

Co-authored-by: Josh Hunt <joshhunt@users.noreply.github.com>

Commit:
[release-12.2.1] Page limit config for dashboards with visible annotations (#111379)

Page limit config for dashboards with visible annotations (#110911)

* Page limit config for dashboards with visible annotations



---------


(cherry picked from commit 77fa3333e4a156a953157a46c34fc4d59d286da1)

Signed-off-by: Maicon Costa <maiconscosta@gmail.com>
Co-authored-by: maicon <maiconscosta@gmail.com>

Commit:
[release-12.2.1] Auth: Add SCIM settings permission to auth config writer role (#111398)

Auth: Add SCIM settings permission to auth config writer role (#111326)

* Auth: add SCIM settings permission to authentication config writer role

* make update-workspace

(cherry picked from commit 1ef27e97499d1c3757051cfe36dbcd318743b3b6)

Co-authored-by: colin-stuart <colindonstuart@gmail.com>

Commit:
[release-12.2.1] Dashboards: Fix missing Ctrl+O keyboard shortcut for crosshair toggle (#111402)

Dashboards: Fix missing Ctrl+O keyboard shortcut for crosshair toggle (#111310)

* Dashboard Scenes: Fix missing Ctrl+O keyboard shortcut for crosshair toggle

- Add missing mod+o keybind to dashboard scenes keyboard shortcuts
- Implement crosshair state cycling (Default -> Crosshair -> Tooltip -> Default)
- Add comprehensive unit tests for keyboard shortcuts functionality
- Add e2e test to verify shortcut works and prevents browser file dialog
- Fix ensures parity between legacy and scenes dashboard implementations

Fixes issue where Ctrl+O/Cmd+O was opening browser file dialog instead of
toggling shared crosshair modes in scenes-based dashboards.

* Remove waitForTimeout from e2e test

- Replace arbitrary timeouts with proper element waiting
- Use waitFor with visible state instead of setTimeout
- Improve test reliability and follow Playwright best practices

* Optimize e2e test for crosshair keyboard shortcut

- Remove unnecessary console logging and timeout settings
- Simplify assertions to only check the currently selected radio button
- Improve test performance by reducing DOM queries
- Focus on essential functionality verification

* Fix linting

(cherry picked from commit c0ce4ff1f2f265730af01b01fe37f67c239fca63)

Co-authored-by: Ivan Ortega Alba <ivanortegaalba@gmail.com>

joshhunt committed:
[release-12.2.1] CI: Backport release-npm.yml (#111396)

backport release-npm.yml from main

Commit:
Release: Bump version to 12.2.1 (#111415)

bump version 12.2.1

Co-authored-by: grafana-delivery-bot[bot] <grafana-delivery-bot[bot]@users.noreply.github.com>

jtvdez committed:
Docs: What's new and Upgrade guide v12.2 (#110727) (#111472)

Commit:
[release-12.2.1] docs: reorder whats new posts and include plugin translation (#111484)

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

kminehart committed:
update release-npm and validate script

kminehart committed:
update missing npm publish scripts

Commit:
[release-12.2.1] Chore: Fix @grafana/alerting package repo (#111532)

Chore: Fix @grafana/alerting package repo (#110939)

(cherry picked from commit ab3c93b279014ced69cabcf0e7a6bfea3e9615d5)

Co-authored-by: Josh Hunt <joshhunt@users.noreply.github.com>

Commit:
[release-12.2.1] Fix grafanactl resource pull command syntax (#111608)

Co-authored-by: Irene Rodríguez <irene.rodriguez@grafana.com>
Fix grafanactl resource pull command syntax (#111465)

Commit:
[release-12.2.1] DOCS: FIx broken link on RBAC for app plugins page (#111633)

DOCS: FIx broken link on RBAC for app plugins page (#111574)

fixed incorrect RefURI

(cherry picked from commit 0bf37742f1d6d55d4de4865081be63dbad32749f)

Co-authored-by: Larissa Wandzura <126723338+lwandz13@users.noreply.github.com>

Commit:
[release-12.2.1] Docs: Update saved queries feature availability details (#111644)

Co-authored-by: Isabel Matwawana <76437239+imatwawana@users.noreply.github.com>

Commit:
[release-12.2.1] FlameGraph: Ensure total is only counted once for recursive function calls (#111606)

FlameGraph: Ensure total is only counted once for recursive function calls (#111548)

grafana-flamegraph: Ensure total is only counted once for recursive function calls

Example flamegraph: https://flamegraph.com/share/2bb59df3-9930-11f0-94ec-760777e76ccd

(cherry picked from commit c5f6318b7b82e3e89e2ba1f583e3695cc40c2db7)

Co-authored-by: Christian Simon <simon@swine.de>

Commit:
[release-12.2.1] Docs: Fix link reference in Generic OAuth documentation (#111656)

Co-authored-by: Andrew Hackmann <5140848+bossinc@users.noreply.github.com>
Fix link reference in Generic OAuth documentation (#111647)

Commit:
[release-12.2.1] Docs: Marking the image renderer plugin as deprecated (#111670)

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

jtroy committed:
[release-12.2.1] Docs: Fix broken links in SAML docs (#111693)

Docs: Fix broken links in SAML docs (#111039)

* Rename this heading to match the link in 'Request Initiation'

* Fix link to 'Configure SAML using the Grafana configuration file' and make the link text match

(cherry picked from commit 004f30fcb7b8bbb68130967bd41b084058cb7cc1)

Commit:
[release-12.2.1] Refactor SCIM provisioning documentation (#111731)

Co-authored-by: Irene Rodríguez <irene.rodriguez@grafana.com>

Commit:
[release-12.2.1] Docs: Edits to OaC/Git Sync (#111732)

Co-authored-by: Anna Urbiztondo <anna.urbiztondo@grafana.com>
Co-authored-by: Irene Rodriguez <irene.rodriguez@grafana.com>

Commit:
[release-12.2.1] Docs: provisioning fix (#111749)

Co-authored-by: Anna Urbiztondo <anna.urbiztondo@grafana.com>
fix (#111744)

Commit:
[release-12.2.1] Documentation Node graph broken link (#111757)

Co-authored-by: Jara Suárez de Puga García <jara.suarezdepuga@grafana.com>
Co-authored-by: Isabel Matwawana <isabel.matwawana@grafana.com>

Commit:
[release-12.2.1] feat(ci): Allow overriding runs-on for publish-artifact workflow (#111765)

feat(ci): Allow overriding runs-on for publish-artifact workflow (#111695)


(cherry picked from commit ffe85d7c7ed2d682aa823e635e60675a42d8537b)

Signed-off-by: Dave Henderson <dave.henderson@grafana.com>
Co-authored-by: Dave Henderson <dave.henderson@grafana.com>

Commit:
[release-12.2.1] Geomap: Fix duplicated data links (#111772)

Commit:
[release-12.2.1] Plugins: Dependencies do not inherit parent URL for preinstall (#111769)

Plugins: Dependencies do not inherit parent URL for preinstall (#111762)

dependencies dont inehrit parent url for preinstall

(cherry picked from commit 073338ec295bf26eede4dba42906c348a0aa71bf)

Co-authored-by: Will Browne <wbrowne@users.noreply.github.com>

Commit:
[release-12.2.1] LDAP Authentication: Fix URL to propagate username context as parameter (#111849)

LDAP Authentication: Fix URL to propagate username context as parameter (#111723)

Fix URL to propagate username context as parameter

(cherry picked from commit 7055ba914024a2bb9daa8421cd2b8ae715cd5afd)

Co-authored-by: Bradley <12028233+bradleypettit@users.noreply.github.com>

Commit:
[release-12.2.1] [InfluxDB]: Remove version notices (#111850)

Commit:
[release-12.2.1] Doc: Filter by value update (#111886)

* Doc: Filter by value update (#111881)

doc(transformation): filter by value

---------

Co-authored-by: Ihor Yeromin <yeryomin.igor@gmail.com>

Commit:
[release-12.2.1] Docs: Add panel options section to dashboard settings (#111947)

Co-authored-by: Isabel Matwawana <76437239+imatwawana@users.noreply.github.com>

Commit:
[release-12.2.1] Chore: Update on-prem stable release dates (#111953)

Chore: Update on-prem stable release dates (#111909)

* baldm0mma/ update release dates

* Update docs/sources/upgrade-guide/when-to-upgrade/index.md



---------


(cherry picked from commit 2f3c5031827fd0ddb67ee6402f0729a75b02a902)

Co-authored-by: Jev Forsberg <46619047+baldm0mma@users.noreply.github.com>
Co-authored-by: Isabel Matwawana <76437239+imatwawana@users.noreply.github.com>

ppcano committed:
[release-12.2.1] docs(alerting): `alertingSaveStateCompressed` is enabled by default (#111972)

* docs(alerting): `alertingSaveStateCompressed` is enabled by default (#111897)

(cherry picked from commit 7ed46fd32138e98c617cab8e949262a3867fc0cf)

* update feature flag default state

Commit:
[release-12.2.1] docs(alerting): add settings for alert evaluation backoff retries (#111973)

docs(alerting): add settings for alert evaluation backoff retries (#111891)

* docs(alerting): add settings for alert evaluation backoff retries

* docs(alerting): add mention of backoff settings on the Error state docs

* fix vale prose

(cherry picked from commit 6248971d1e3a225bbad6aca127e6ae3b18c6a319)

Co-authored-by: Pepe Cano <825430+ppcano@users.noreply.github.com>

Commit:
[release-12.2.1] Influx: Improve variable regex handling (#111941)

Influx: Improve variable regex handling (#111853)

(cherry picked from commit b50737a76ceaa301d07deda42b8336e20b6d7953)

Co-authored-by: Andreas Christou <andreas.christou@grafana.com>

joshhunt committed:
[release-12.2.1] NPM: Backport NPM publishing from main (#112000)

* Backport npm publishing workflow from main

* Ignore false-cjs in validate-npm-packages

(cherry picked from commit 6916b3943955947c75a6f031e050384ce5e5896a)
(cherry picked from commit 1aba469bab413628f61faeddd30e3de5c1820b7a)

* Ignore untyped-resolutions for grafana-i18n in validate-npm-packages

(cherry picked from commit b250fe8d22274a5427d4747ab8736e325150f18f)

jtvdez committed:
[release-12.2.1] Update index.md (#112028) (#112064)

Co-authored-by: Michael Lamb <michael@michaellamb.dev>

Commit:
[release-12.2.1] Docs: OaC/Git Sync updates for Obs Con (#112066)

Co-authored-by: Anna Urbiztondo <anna.urbiztondo@grafana.com>
Co-authored-by: Jack Baldry <jack.baldry@grafana.com>

Commit:
[release-12.2.1] Docs: Fixed a broken link on the Intro to Exemplars page (#112068)

Co-authored-by: Larissa Wandzura <126723338+lwandz13@users.noreply.github.com>
Co-authored-by: Irene Rodríguez <irene.rodriguez@grafana.com>
Fixed a broken link on the Intro to Exemplars page (#112025)

Commit:
[release-12.2.1] Prometheus: Prom Azure and AWS auth with mig info (#112072)

Prometheus: Prom Azure and AWS auth with mig info (#111142)

* docs for prom type mig

* andreas feedback

* update scripts and add forward settings issue

* make links internal

* prettier

* Update docs/sources/datasources/prometheus/configure/aws-authentication.md



* Update docs/sources/datasources/prometheus/configure/aws-authentication.md



* Apply suggestions from code review



* Update docs/sources/datasources/prometheus/configure/aws-authentication.md

* Apply suggestions from code review

* Update docs/sources/datasources/prometheus/configure/azure-authentication.md

* feedback

* prettier

* clarify the revert script is only for self-hosted

* Update docs/sources/datasources/prometheus/configure/aws-authentication.md



* Apply suggestions from code review



---------


(cherry picked from commit 96c24bc64ce18786c21ec60edfec6101987a07e9)

Co-authored-by: Andrew Hackmann <5140848+bossinc@users.noreply.github.com>
Co-authored-by: Larissa Wandzura <126723338+lwandz13@users.noreply.github.com>

Commit:
[release-12.2.1] Docs: Git Sync - Tim's feedback (#112104)

Docs: Git Sync - Tim's feedback (#112098)

* Feedback

* Prettier

(cherry picked from commit 96b5f63711221b98198943b0d6998d2b70697558)

Co-authored-by: Anna Urbiztondo <anna.urbiztondo@grafana.com>

Commit:
[release-12.2.1] Docs: Last note for GitSync for ObsCon (#112085)

Co-authored-by: Anna Urbiztondo <anna.urbiztondo@grafana.com>

fastfrwrd committed:
[release-12.2.1] Table: Backport the Safari 26 fixes to 12.2.1 (#111906)

* [release-12.2.1] Table: Disable virtualization, hover overflow, and scrollbar width resizing on Safari 26 (#111834)

* Table: Disable virtualization, hover overflow, and scrollbar width resizing on Safari 26

* pull obj def out of method

* [release-12.2.1] Table: Avoid overflow issues in Safari 26 (#111858)

Commit:
[release-12.2.1] DOCS: Amazon CloudWatch data source docs revmp (#112132)

DOCS: Amazon CloudWatch data source docs revmp (#109945)

* started new configure doc

* updates to the configure doc

* worked on variables doc

* made edits

* query editor edits

* query editor updates

* continued with updates

* added updates

* additional updates, rewrite of auth doc

* finished query editor updates

* cleaned up the intro doc, added ref URIs to other docs

* some final edits

* more edits prior to PR creation

* ran prettier, added a screenshot

* linter fixes

* updates based on questions doc

* ran prettier

* updates based on feedback

* made more edits based on feedback

* more updates based on feedback

* updates based on feedback

* moved alerting under logs

* ran prettier

(cherry picked from commit 725a91e9eb77f6ed2f5caf1fb1a26d256988c1bd)

Co-authored-by: Larissa Wandzura <126723338+lwandz13@users.noreply.github.com>

ppcano committed:
[release-12.2.1] docs(alerting): Email contact point enhancements (#112121)

docs(alerting): Email contact point enhancements (#111944)

* docs(alerting): Email contact point enhancements

* minor content tweaks

(cherry picked from commit 5438df01a1a95a345bc8dab85980f653529b539c)

macabu committed:
[release-12.2.1] Go: Update to 1.25.2 + golangci-lint v2.5.0 + golang.org/x/net v0.45.0 (#112156)

* Go: Update to 1.25.2

* golangci-lint: Update to 2.5.0

* Dependencies: Bump golang.org/x/net to v0.45.0

* Fix broken feature toggle test

Commit:
[release-12.2.1] Docs: Clarify RBAC availability in documentation (#112173)

Co-authored-by: Irene Rodríguez <irene.rodriguez@grafana.com>

Commit:
[release-12.2.1] Explore: Add missing height to logs in Explore (#112177)

Explore: Add missing height to logs in Explore (#112176)

* Explore: Add missing height to logs in Explore

* Add max height to controlled log rows

(cherry picked from commit a0fc683a67ef9b73f20934f910c3c2b450451f75)

Co-authored-by: Matias Chomicki <matias.chomicki@grafana.com>

Commit:
[release-12.2.1] Auth: Fix render user OAuth passthrough (#112092)

Auth: Fix render user OAuth passthrough (#111636)

* devenv: fix volumes section when sources don't contain one

* wip

* Working correctly with improvedExternalSessionHandling on

* Remove not needed lines

* Working with the old flow, tests

* Handle compatibility with the feature toggle, tests wip

* Tests

* Cleanup

* Address feedback

* Align tests

* Add comment

* Fix issue with session removal after the invalidation of tokens

* Remove commented out code

* clean up

(cherry picked from commit 53f4803e98fd86e3eccd67456620cdd7a3856a25)

Co-authored-by: Misi <mgyongyosi@users.noreply.github.com>

Commit:
[release-12.2.1] Docs: Adding new entry to 12.2 whats new (#112235)

Docs: Adding new entry to 12.2 whats new (#112234)

(cherry picked from commit 5eb295d8501cc80986ffed74e22bdae666fbc204)

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

Commit:
[release-12.2.1] Docs: Clarify role assignment to users and teams when using terraform (#112261)

Co-authored-by: Vardan Torosyan <vardants@gmail.com>

Commit:
[release-12.2.1] GenAI: Fix uncaught error when panel title is missing (#112286)

Commit:
[release-12.2.1] TimeSeries: Fix memory leak with boolean fields (#112290)

TimeSeries: Fix memory leak with boolean fields (#112252)


(cherry picked from commit 248c20a6b681e188e30fe1dbdbc5f3d997bbb349)

Co-authored-by: Leon Sorokin <leeoniya@gmail.com>
Co-authored-by: Paul Marbach <paul.marbach@grafana.com>

Commit:
[release-12.2.1] Logs context: fix position on small viewports (#112343)

Logs context: fix position on small viewports (#112342)

(cherry picked from commit 6c726cf0ea84c1cd08eee3f85b659176120ab0dd)

Co-authored-by: Matias Chomicki <matias.chomicki@grafana.com>

macabu committed:
[release-12.2.1] Go: Update to 1.25.3 (#112361)

Go: Update to 1.25.3

Proximyst committed:
[release-12.2.1] Chore: Update Redis library to v9 (#112363)

Commit:
[release-12.2.1] Docs: Add batch_wait_duration and batch_size_bytes to usage insights Loki exporter (#112416)

Docs: Add batch_wait_duration and batch_size_bytes to usage insights Loki exporter (#112408)

* Docs: Add batch_wait_duration and batch_size_bytes to usage insights Loki exporter

* Run linter

(cherry picked from commit 1c614fdc9a8873b2d42c29f005e69fad2cbbb5aa)

Co-authored-by: Matheus Macabu <macabu@users.noreply.github.com>

Commit:
[release-12.2.1] Docs: Manage multi-team access (#112500)

Co-authored-by: Anna Urbiztondo <anna.urbiztondo@grafana.com>
Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>
Co-authored-by: Irene Rodríguez <irene.rodriguez@grafana.com>
Co-authored-by: Sarah Constant <sarahleejane@users.noreply.github.com>

Commit:
[release-12.2.1] Grammar: wrap in try/catch in case regex creation fails (#112600)

Grammar: wrap in try/catch in case regex creation fails (#112595)

* Grammar: wrap in try/catch in case regex creation fails

* Improve breadcrumbs

(cherry picked from commit bbb5322008d814978bcaa0322d171b7af37371a4)

Co-authored-by: Matias Chomicki <matias.chomicki@grafana.com>

Commit:
[release-12.2.1] Docs: Git Sync resource reqs (#112592)

Co-authored-by: Anna Urbiztondo <anna.urbiztondo@grafana.com>
Co-authored-by: Irene Rodríguez <irene.rodriguez@grafana.com>

Commit:
[release-12.2.1] docs(alerting) Add examples of high-cardinality alerts (#112633)

docs(alerting) Add examples of high-cardinality alerts (#112311)

* docs(alerting) Add examples of high-cardinality alerts

* minor intro edits

(cherry picked from commit 9e505ea2dedbfd73ef4421c2d91cf7391922f75d)

Co-authored-by: Pepe Cano <825430+ppcano@users.noreply.github.com>

linoman committed:
[release-12.2.1] Backport 112615 to release 12.2.1 (#112636)

Update validation of non-provisioned users rejection (#112615)

* Update validation of non-provisioned users rejection

* Align tests

(cherry picked from commit 0e4237b77586f585a290a8d450d341a6c88143be)

joshhunt committed:
[release-12.2.1] CI: Remove OIDC debug step from npm publish (#112657)

CI: Remove OIDC debug step from npm publish (#112631)

(cherry picked from commit b0acfd1189f8f63a19c1d746829b7fd8f63383da)

mgyongyosi commented on docs/sources/setup-grafana/configure-access/configure-authentication/saml/_index.md:
```diff
@@ -31,14 +31,14 @@ You can configure SAML authentication in Grafana through one of the following me
-- [Configure SAML with Azure AD](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-security/configure-authentication/saml/configure-saml-with-azuread/)
-- [Configure SAML with Okta](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-security/configure-authentication/saml/configure-saml-with-okta/)
-- [Configure SAML with Okta catalog application](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-security/configure-authentication/saml/configure-saml-with-okta/oin-application)
+- [Configure SAML with Azure AD](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-access/configure-authentication/saml/configure-saml-with-azuread/)
```
```suggestion
- [Configure SAML with Entra ID](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-access/configure-authentication/saml/configure-saml-with-azuread/)
```

mgyongyosi commented on docs/sources/setup-grafana/configure-access/configure-authentication/saml/_index.md:
```diff
@@ -181,7 +181,7 @@ By default, new Grafana users using SAML authentication will have an account cre
 
 1.  **Ensure Consistent Identifier in SAML Assertion:**
-    - The unique identifier from Azure AD (typically `user.objectid`) that you mapped to the `externalId` attribute in Grafana in your SCIM provisioning setup **must also be sent as a claim in the SAML assertion.** For more details on SCIM, refer to the [SCIM provisioning documentation](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-security/configure-scim-provisioning/).
+    - The unique identifier from Azure AD (typically `user.objectid`) that you mapped to the `externalId` attribute in Grafana in your SCIM provisioning setup **must also be sent as a claim in the SAML assertion.** For more details on SCIM, refer to the [SCIM provisioning documentation](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-access/configure-authentication/).
```
```suggestion
    - The unique identifier from Entra ID (typically `user.objectid`) that you mapped to the `externalId` attribute in Grafana in your SCIM provisioning setup **must also be sent as a claim in the SAML assertion.** For more details on SCIM, refer to the [SCIM provisioning documentation](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-access/configure-authentication/).
```

IevaVasiljeva reviewed (approved):
Awesome, thanks for making the change! I like this layout much more than the previous one.

github-actions[bot] committed:
Release: 12.2.1 (#112736)

* Update changelog

* Update version to 12.2.1

---------

Co-authored-by: grafana-delivery-bot[bot] <grafana-delivery-bot[bot]@users.noreply.github.com>

Commit:
[release-12.2.1] Tag / content discrepancy (#111889)

Co-authored-by: Clément Duveau <clement@duveau.eu>

Commit:
[release-12.2.1] Include author in patch creation (#112696)

Include author in patch creation (#112675)

Include author in security mirror

(cherry picked from commit ef2e62c8522d5a61ec8038786ea2c5cb9f5c46f7)

Co-authored-by: Kevin Minehart <5140827+kminehart@users.noreply.github.com>

Commit:
[release-12.2.1] Alerting: Fix unmarshalling of GettableStatus to include time intervals (#112734)

Alerting: Fix unmarshalling of GettableStatus to include time intervals (#112602)

* move test files into test-data

* add test for the bug

* populate time-intervals of gettableStatus config

(cherry picked from commit 5f9a51418c5bbf460d54741bcab0f00ae3a4e616)

Co-authored-by: Yuri Tseretyan <yuriy.tseretyan@grafana.com>

Commit:
[release-12.2.2] docs(alerting): clarify usage of templates in webhook custom payloads (#112755)

docs(alerting): clarify usage of templates in webhook custom payloads (#112672)

* docs(alerting): clarify usage of templates in webhook custom payloads

* Update docs/sources/alerting/configure-notifications/template-notifications/manage-notification-templates.md



---------


(cherry picked from commit fb5c5411f86d6e258a9c22308fa0a707c20b9fee)

Co-authored-by: Pepe Cano <825430+ppcano@users.noreply.github.com>
Co-authored-by: Johnny Kartheiser <140559259+JohnnyK-Grafana@users.noreply.github.com>

Commit:
[release-12.2.2] Docs: Added known limitations to SQL Expressions (#112760)

Docs: Added known limitations to SQL Expressions (#112676)

* initial new section creation

* added additional known limitations

* adding some clarification

(cherry picked from commit e5627bcc6761ccfabc5b06bb3cf609a3f33ca9fc)

Co-authored-by: Larissa Wandzura <126723338+lwandz13@users.noreply.github.com>

jdbaldry committed:
Merge remote-tracking branch 'origin/master' into jdb/2025-10-iam-restructure

jdbaldry committed:
Fixing links

jdbaldry committed:
Revert "Fixing links"

This reverts commit 372564bb40dd4c1ff390d4e46cc36bab87329c77.

jdbaldry committed:
Apply to 12.2

Signed-off-by: Jack Baldry <jack.baldry@grafana.com>

jdbaldry committed:
Handle rejected hunks

Signed-off-by: Jack Baldry <jack.baldry@grafana.com>

jdbaldry committed:
Merge remote-tracking branch 'origin/master' into jdb/2025-10-iam-restructure
